### PR TITLE
Update PoR Wallet Address cross-chain reserves description

### PR DESCRIPTION
**Pull request by dwightjl**

### CODE REVIEW

**Review by rui-zhang89 — COMMENTED** [review]
Can we change the "attest" wording to "report" instead? 
- The project attests which addresses they own through a self-hosted API.
- Chainlink is not responsible for the accuracy of self-attested reserves data.

File: src/content/data-feeds/proof-of-reserve/index.mdx [hunk 1]
```diff
@@ -45,7 +45,7 @@ Cross-chain reserves are sourced from the network where the reserves are held. C
 Cross-chain reserves provide their data using the following methods:
 
 - Wallet address manager: The project uses the [IPoRAddressList](https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.8/interfaces/PoRAddressList.sol) wallet address manager contract and self-attests to which addresses they own.
-- Wallet address: The project attests which addresses they own through a self-hosted API.
+- Wallet address: The project reports which addresses they own through a self-hosted API. Reserve data reported by asset issuer's self-hosted API carries additional risks. Chainlink is not responsible for the accuracy of self-reported reserves data. Users must do their own risk assessment for asset issuer risk.
```
Comment (aelmanaa) [inline src/content/data-feeds/proof-of-reserve/index.mdx:48]:
```suggestion
- Wallet address: The project reports which addresses they own through a self-hosted API. Reserve data reported by asset issuer's self-hosted API carries additional risks. Chainlink Labs is not responsible for the accuracy of self-reported reserves data. Users must do their own risk assessment for asset issuer risk.
```

**Review by aelmanaa — DISMISSED** [review]

File: src/content/data-feeds/proof-of-reserve/index.mdx [hunk 2]
```diff
@@ -44,8 +44,8 @@ Cross-chain reserves are sourced from the network where the reserves are held. C
 
 Cross-chain reserves provide their data using the following methods:
 
-- Wallet address manager: The project uses the [IPoRAddressList](https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.8/interfaces/PoRAddressList.sol) wallet address manager contract and self-attests to which addresses they own.
-- Wallet address: The project attests which addresses they own through a self-hosted API.
+- Wallet address manager: The project uses the [IPoRAddressList](https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.8/interfaces/PoRAddressList.sol) wallet address manager contract and self-reports which addresses they own.
```
Comment (rui-zhang89) [inline src/content/data-feeds/proof-of-reserve/index.mdx:47]: Append this disclaimer at the end of the sentence as well:

> Reserve data reported by an asset issuer's self-hosted API carries additional risks. Chainlink Labs is not responsible for the accuracy of self-reported reserves data. Users must do their own risk assessment for asset issuer risk.

File: src/content/data-feeds/proof-of-reserve/index.mdx [hunk 3]
```diff
@@ -32,9 +32,9 @@ Offchain reserves are sourced from APIs through an [external adapter](/chainlink
 
 Offchain reserves provide their data using the following methods:
 
-- Third-party: An auditor, accounting firm, or other third party attests to reserves. This is done by combining both fiat and investment assets into a numeric value that is reported against the token.
+- Third-party: An auditor, accounting firm, or other third party audits and verifies reserves. This is done by combining both fiat and investment assets into a numeric value that is reported against the token.
 - Custodian: Reserves data are pulled directly from the bank or custodian. The custodian has direct access to the bank or vault holding the assets. Generally, this works when the underlying asset pulled requires no additional valuation and is simply reported onchain.
-- ⚠️ Self-attested: Reserve data is read from an API that the token issuer hosts. Self-attested feeds carry additional risk.
+- ⚠️ Self-reported: Reserve data is read from an API that the token issuer hosts. Self-reported feeds carry additional risk.
```
Comment (rui-zhang89) [inline src/content/data-feeds/proof-of-reserve/index.mdx:37]: Append this disclaimer at the end of the sentence as well:

> Reserve data reported by an asset issuer's self-hosted API carries additional risks. Chainlink Labs is not responsible for the accuracy of self-reported reserves data. Users must do their own risk assessment for asset issuer risk.

File: src/features/feeds/callouts/por-wallet-address-manager.mdx [hunk 4]
```diff
@@ -1,3 +1,3 @@
...
+Cross-chain Proof of Reserve feeds using wallet address manager contracts can vary in their configurations. If the wallet address manager is self-reporting, these feeds might include reserve balances from onchain addresses that have not been cryptographically verified to show ownership or control. In circumstances where cryptographically verified ownership is not provided, there is a risk that token issuers could manipulate the value of a Proof of Reserve feed by adding more addresses to the address list even if they do not directly control these addresses. Each feed’s specific configuration is documented in the contract address section.
```
Comment (rui-zhang89) [inline src/features/feeds/callouts/por-wallet-address-manager.mdx:3]: Append this disclaimer at the end of the sentence as well:

> Chainlink Labs is not responsible for the accuracy of self-reported reserves data. Users must do their own risk assessment for asset issuer risk.

**Review by rui-zhang89 — CHANGES_REQUESTED** [review]

**Review by rui-zhang89 — APPROVED** [review]

**Pull request merged**